canonical-websites/build.snapcraft.io
# Add filtering repos to 'Add repos' page

## Done

- Creates filter state and component in the repos list
- Applies filter state when rendering the repo list
- Resolves a few styling issues for the search component

## QA

- Check out this feature branch
- Run the site using the command ```npm start -- --env=environments/dev.env```
- View the site locally in your web browser at: [http://0.0.0.0:8000/](http://0.0.0.0:8000/)
- Going to the add repos listing, filter the list by entering a username, organisation or repo and the list should filter automatically.


## Issue / Card

#867
